tests/test_brizo.py

```diff
@@ -87,6 +96,17 @@ def get_registered_ddo(account, providers=None):
 
     ddo.add_authentication(did, PUBLIC_KEY_TYPE_RSA)
 
+    try:
+        _oldddo = aqua.get_asset_ddo(ddo.did)
+        if _oldddo:
+            aqua.retire_asset_ddo(ddo.did)
+    except ValueError:
+        pass
+
+    if not plecos.is_valid_dict_local(ddo.metadata):
+        print(f'invalid metadata: {plecos.validate_dict_local(ddo.metadata)}')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [invalid syntax](https://app.codacy.com/app/ocean-protocol/brizo/pullRequest?prid=4502265)

Reply:
In Python >= v3.6 this is valid.

Reply:
Cool, @ttmc is just a bot now :-)

On Codacy, PyLint is setup for Python 2, I activated it for Python3 but no idea what settings to set https://app.codacy.com/p/174333/patterns/list?engine=31677b6d-4ae0-4f56-8041-606a8d7a8e61

So guess all Python projects should be checked with PyLint for Python 3 then